server-kotlin/build.gradle

```diff
@@ -1,51 +1,48 @@
 buildscript {
     repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.github.jengelman.gradle.plugins:shadow:5.2.0'
+        mavenCentral()
     }
 }
 
 plugins {
+    id 'org.springframework.boot' version "2.2.5.RELEASE"
+    id 'io.spring.dependency-management' version "1.0.9.RELEASE"
+    id 'org.jetbrains.kotlin.jvm' version "$kotlin_version"
+    id 'org.jetbrains.kotlin.plugin.spring' version "$kotlin_version"
     id 'com.google.cloud.tools.jib' version '2.1.0'
 }
 
-apply plugin: 'kotlin'
-apply plugin: 'idea'
-apply plugin: 'application'
-apply plugin: 'com.github.johnrengelman.shadow'
 
 group 'digital.cesko'
-version '0.0.4'
-mainClassName = "io.ktor.server.netty.EngineMain"
+version '0.0.5'
+java.sourceCompatibility = JavaVersion.VERSION_11
 
 repositories {
-    mavenLocal()
+    mavenCentral()
     jcenter()
-    maven { url 'https://kotlin.bintray.com/ktor' }
 }
 
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    compile "io.ktor:ktor-server-netty:$ktor_version"
-    compile "ch.qos.logback:logback-classic:$logback_version"
-    compile "io.ktor:ktor-server-core:$ktor_version"
-    compile "io.ktor:ktor-auth:$ktor_version"
-    compile "io.ktor:ktor-auth-jwt:$ktor_version"
-    compile "io.ktor:ktor-jackson:$ktor_version"
-    compile "io.ktor:ktor-client-json-jvm:$ktor_version"
-    compile "io.ktor:ktor-client-jackson:$ktor_version"
-    compile "io.ktor:ktor-client-apache:$ktor_version"
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    compile "org.springframework.boot:spring-boot-starter-actuator"
```

Comment:
you should use `implementation` for all dependencies I guess

Reply:
Done, thanks